R/packages.R

```diff
@@ -103,7 +99,7 @@ get_github_wasm_assets <- function(desc) {
 
   # Find GH release asset URLs for R library VFS image
   library_data <- Filter(function(item) {
-    item$name == "library.data"
+    grepl("library.data", item$name)
```

Comment:
Note: This change is so that compressed `library.data.gz` packages are also picked up from GitHub releases. GitHub releases use the older Emscripten-style `.data`/`.metadata` VFS format since they can contain a full R library of multiple R packages.